modules/core/src/test/scala/gem/enum/Arbitraries.scala

```diff
@@ -1,69 +0,0 @@
-// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
-// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
-
-package gem.enum
-
-import org.scalacheck._
-import org.scalacheck.Gen._
-
-
-trait Arbitraries {
-
-  private def pick[A](as: List[A]): Arbitrary[A] =
-    Arbitrary(oneOf(as))
-
-
-  // Flamingos2
-
-  implicit val arbF2Disperser   = pick(F2Disperser.all  )
```

Comment:
Do we still get arbitraries for the instrument enums somewhere?  I'm not seeing them.